zhaquirks/tuya/ts0601_pirmotion.py

```diff
@@ -0,0 +1,140 @@
+"""BlitzWolf IS-3/Tuya motion rechargeable occupancy sensor."""
+
+import math
+from typing import Dict
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Ota, Time
+from zigpy.zcl.clusters.measurement import IlluminanceMeasurement, OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import TuyaLocalCluster
+from zhaquirks.tuya.mcu import (
+    DPToAttributeMapping,
+    TuyaMCUCluster,
+    TuyaPowerConfigurationCluster,
+)
+
+
+class TuyaOccupancySensing(OccupancySensing, TuyaLocalCluster):
+    """Tuya local OccupancySensing cluster."""
+
+
+class TuyaIlluminanceMeasurement(IlluminanceMeasurement, TuyaLocalCluster):
+    """Tuya local IlluminanceMeasurement cluster."""
+
+
+class SensitivityLevel(t.enum8):
+    """Sensitivity level enum."""
+
+    LOW = 0x00
+    MEDIUM = 0x01
+    HIGH = 0x02
+
+
+class OnTimeValues(t.enum8):
+    """Sensitivity level enum."""
+
+    _10_SEC = 0x00
+    _30_SEC = 0x01
+    _60_SEC = 0x02
+    _120_SEC = 0x03
+
+
+class PirMotionManufCluster(TuyaMCUCluster):
+    """Neo manufacturer cluster."""
```

Comment:
Should this comment be updated?

Reply:
It currently matches the comment in the ts0601_motion.py. If you have a preference, please let me know and I will update it.